### ISSUE TITLE
GaussianNoise support

### ISSUE THREAD

**Issue by boris-kuz** [Dobiasd/frugally-deep#169]
Hi!

I was wondering what the reason to not support GaussianNoise layers is?
It's a regularization layer, so it's only active during training (https://keras.io/layers/noise/), i.e. it should have no impact on the prediction whatsoever, right?
Is there a way to trivially implement support for networks using GaussianNoise layers, e.g. by ignoring them during the loading of the model or something?

Thanks!

**Comment by Dobiasd** [comment]
Hi Boris,

thanks for the suggestion.

I guess if `GaussianNoise` is just an identity function during prediction, it should be trivial to add support for it to frugally-deep, like already has been done with the dropout layers (see [here](https://github.com/Dobiasd/frugally-deep/blob/6b62d34537bc69bae96eaae6fa00017ed52ade59/include/fdeep/import_model.hpp#L498-L504) and [here](https://github.com/Dobiasd/frugally-deep/blob/6b62d34537bc69bae96eaae6fa00017ed52ade59/include/fdeep/import_model.hpp#L1082))

Would you be interested in giving it a try and create a pull request?

**Comment by boris-kuz** [comment]
Oh, that looks fairly straight forward! I'll give it a try and submit a pull request, probably later in the day. Thank you!

**Comment by Dobiasd** [comment]
Thanks for the quick PR. :+1:

**Comment by boris-kuz** [comment]
No worries :)
